pkg/flowaggregator/clickhouseclient/clickhouseclient.go

```diff
@@ -487,24 +496,33 @@ func (ch *ClickHouseExportProcess) batchCommitAll() (int, error) {
 
 		if err != nil {
 			klog.ErrorS(err, "Error when adding record")
+			ch.pushFrontRecords(recordsToExport)
```

Comment:
I think the new implementation makes sense, given that the likelihood of an error is small